tests/testTriggers.go

```diff
@@ -595,13 +595,14 @@ func TriggerDeployNewApps(contexts *[]*scheduler.Context, recordChan *chan *Even
 		for _, ctx := range *contexts {
 			log.Infof("Validating context: %v", ctx.App.Key)
 			ctx.SkipVolumeValidation = false
+			errorChan = make(chan error, errorChannelSize)
+			go func() {
```

Comment:
why are you running this in routine?

Reply:
As using range on a channel will block the execution I had to remove the for loop after the ValidateContext function because it closes the errorChan. Although I could have placed ValidateContext inside a goroutine, I chose this approach instead.

Reply:
My bad. I thought we could not iterate over a closed channel and made the changes. Will revert it.

Reply:
Done.